src/test/java/spoon/test/model/TypeTest.java

```diff
@@ -61,24 +63,23 @@ public void testGetDifferentElements() throws Exception {
 	}
 
 	@Test
+	@DisabledForJreRange(min = JRE.JAVA_14)
+	/*
+	* This annotation is needed because in java.lang.object the method registerNative was removed.
+	* See https://bugs.openjdk.java.net/browse/JDK-8232801 for details.
+	*/
```

Comment:
Should this be considered a bug (for which we need a bug ticket)?

Reply:
No, we simply tested against a real internal method `private native registerNative()` which was a relict of the sun java time. We should stop testing against these internals which are not really stable.

> In this case we are deleting a private method from the Object class. There are no specification implications here as it is a private method. However there is, as described, a behavioural change, because the now deleted method will no longer be considered during method resolution and so code that would currently generate an error, no longer will.

Reply:
Ah, I misunderstood, I thought it was the production code that used an internal method.